setup.cfg

```diff
@@ -54,4 +54,4 @@ console_scripts =
     tern = tern.__main__:main
 
 [options.extras_require]
-dev = bandit~=1.6; prospector>=1.2; GitPython~=2.1
+dev = bandit~=1.6; prospector>=1.2; GitPython~=2.1; tox
```

Comment:
On my machine, the tox version is 3.14.5 Is that the same for you? If yes, then add `tox>=3.14`